### ISSUE TITLE
I only have 4GB GPU - is it possible to accumulate batches?

### ISSUE THREAD

**Issue by Motherboard** [facebook/fb.resnet.torch#114]
I don't have enough memory for decent batch sizes... is my only option to get a better GPU \ more GPUs?

In Caffe I can define a commulative batch size.

**Comment by colesbury** [comment]
There's no command-line option to accumulate batches. You can modify the code to accumulate the gradients across multiple mini-batches, but it might be better to decrease the learning rate and run with a smaller batch size.

**Comment by Cysu** [comment]
@Motherboard you may refer to my commit [here](https://github.com/Cysu/fb.resnet.torch/commit/53f9fcb4346d6dea720160094f0991030975323f) for a quick implementation of the `iter_size` option like that in caffe. Note that small batch size could make the batch normalization unstable and thus degrade the performance.
